### PR TITLE
Updates on schedule, masking, other information about the Summit

### PR DESCRIPTION
**Pull request by jberkus**
... based on April 25th Meeting.

@puja108 @mrbobbytables @pnbrown @debgiles 

Signed-off-by: Josh Berkus <josh@agliodbs.com>

### CODE REVIEW

File: content/en/events/kcseu/location.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ Registration and badge pick-up will be available at the
 You **MUST** go through vaccination verification and pick up your badge before
 attending the summit or evening social event.
 
-The summit itself will be on Level 3 of the Feria Valencia Event Center.
+The summit itself will be on Level 3 of the Feria Valencia Event Center. Masking
+and vaccination is required to attend.
```
Comment (tokt) [inline content/en/events/kcseu/location.md:16]: /s/is/are

**Review by tokt — COMMENTED** [review]
Thank you!
/lgtm

File: content/en/events/kcseu/location.md (hunk 1 above)
Comment (jberkus) [reply content/en/events/kcseu/location.md:16]: fixed

File: content/en/events/kcseu/faq.md [hunk 2]
```diff
@@ -8,13 +8,15 @@ weight: 4
...
 - [Why is vaccination required to attend in-person?](#why-is-vaccination-required-to-attend-in-person)
+- [Do I need to wear a mask?](#do-i-need-to-wear-a-mask)
 - [Why do I need to be a Kubernetes Org member to attend in-person?](#why-do-i-need-to-be-a-kubernetes-org-member-to-attend-in-person)
 - [What is a sponsored attendee?](#what-is-a-sponsored-attendee)
 - [I am not an Org member. Can I still attend?](#i-am-not-an-org-member-can-i-still-attend)
 - [I have an idea for a session.  How do I propose it?](#i-have-an-idea-for-a-session)
 - [My SIG wants to meet at the Summit](#my-sig-wants-to-meet-at-the-summit)
 - [Will there be a Doc Sprint?](#will-there-be-a-doc-sprint)
 - [Will there be a New Contributor Workshop?](#will-there-be-a-new-contributor-workshop)
+- [Can I bring a guest to the Contributor Socal](#can-i-bring-a-guest-to-the-social)
```
Comment (puja108) [inline content/en/events/kcseu/faq.md:19]:
```suggestion
- [Can I bring a guest to the Contributor Social](#can-i-bring-a-guest-to-the-social)
```

File: content/en/events/kcseu/faq.md [hunk 3]
```diff
@@ -74,7 +79,7 @@ contact us at community@kubernetes.io about registering.
 If you are a SIG lead or subproject owner and there is someone that is not yet
 an org member or from another project that might take part in a panel,
 SIG face-to-face meeting etc, you can sponsor them to attend the summit.
-Please reach out to community@kubernetes.io to make arrangements.
+Please [email us] to make arrangements.
```
Comment (puja108) [inline content/en/events/kcseu/faq.md:82]:
```suggestion
Please [email us](community@kubernetes.io) to make arrangements.
```

Or is there some special kind of markdown replacement in place to set the email?

Comment (jberkus) [reply content/en/events/kcseu/faq.md:82]: Yes, there is, see the bottom of the doc.

File: content/en/events/kcseu/faq.md (hunk 2 above)
Comment (jberkus) [reply content/en/events/kcseu/faq.md]: fixed

**Comment by pnbrown** [comment]
LGTM

**Pull request merged**